guides/git-guidelines.md

```diff
@@ -78,46 +82,45 @@ fix: Users can't logout when authenticated via oauth2
 JIRA Issue: 23456
 ```
 
-## Git branches naming
+## <a name='GitBranchesNaming'></a>Git branches naming
 
 Any branch created for the project will have the following structure:
 
 `<type>/<short-name-of-task>`
 
 Samples:
 
-- feat/users-crud
+- feature/users-crud
 - fix/logout-for-oauth-users
 
+## <a name='GitWorkflow'></a>Git workflow
 
-## Git workflow
-
-We are using a modified and simplified version of Gitflow.
+We are using a modified and simplified version of [Gitflow](https://guides.github.com/introduction/flow/).
 
-For big projects already deployed to production there will always be 2 branches: `master` and `develop`.
-The master branch will contain the code that has been deployed while the develop branch will contain the most recent stable version of it.
+For big projects already deployed to production there will always be 2 branches: `main` and `development`.
+The **main** branch will contain the code that has been deployed while the **development** branch will contain the most recent stable version of it.
 
-For small projects or projects that are have not been deployed we allow simplifying this by working only with a `master` branch.
+For small projects or projects that are have not been deployed we allow simplifying this by working only with a `main` branch.
 
-These are the required steps to add new code to a main branch (develop or master, depending on the nature of the project).
+These are the required steps to add new code to a branch (development or main, depending on the nature of the project).
 
-- Create a new branch using the branch naming convention from the main branch.
-- Do as many commits as are required to complete the feature.
-- You may open a _draft_ Pull Request (PR) if you want some colleague to review your work in progress.
+- Create a new branch using the branch naming convention from the branch you think relevant (main, development).
```

Comment:
Better as an ordered list?

Also, I'm not using MarsBased's gitFlow, but normal feature branches should always start from development, and hotfixes should start from main, isn't it? "Think relevant" is very subjective, whereas gitFlow is quite rigid.

I may be wrong.

Reply:
ordered list, check! ;)

Yes, "think relevant" was with that purpose. I don't know from from where people will open their branches and I was thinking that it could be too much here saying that you should open from this or that branch where it will always depend on the project.

But yes, I agree with you. Open new branches means something new is coming so it should came from development, as this one is the one that always has the latest stable code. Lets see what the guys say tomorrow. Thanks! ;)